app/app.js

```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
   var daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
   $newChicken.attr('data-name', this.title);
   $newChicken.attr('data-startlaying', this.startLaying);
+  $newChicken.attr('data-temper', this.temperament);
   $newChicken.find('h3').text(this.title);
-  $newChicken.find('li:first').text("Egg Production: " + this.eggProduction);
-  $newChicken.find('li:nth-child(2)').text("Temperament: " + this.temperament);
-  $newChicken.find('li:nth-child(3)').text("Starts Laying Eggs By: " + this.startLaying);
-  $newChicken.find('li:last').text("Average Weight: " + this.weight);
+  $newChicken.find('#1').text("Egg Production: " + this.eggProduction);
+  $newChicken.find('#2').text("Temperament: " + this.temperament);
+  $newChicken.find('#3').text("Starts Laying Eggs By: " + this.startLaying);
+  $newChicken.find('#4').text("Average Weight: " + this.weight);
   $newChicken.find('.publishedOn').text("Published on " + this.publishedOn +" (" + daysAgo +" days ago)");
   $newChicken.find('.description').html(this.description);
-  $newChicken.find('.headShot').html(this.image);
+  $newChicken.find('.head-shot img').attr('src',  this.image);
```

Comment:
Watch your eslint to check on correct spacing and indenting... on this line, only one space after "'src',".